docs/installApps.md

```diff
@@ -19,6 +21,7 @@ apply plugin: 'war'
 liberty {
     server {
         name = 'myServer'
+        apps = [file('libertyApp.war'), libertyWarTask]
         dropins = [war]
         stripVersion = true
     }
```

Comment:
update libertyExtensions.md for the apps and dropins entries.  Can also be file type.